WordPress/Resources/AppImages.xcassets/AppIcon-Internal.appiconset/Contents.json

```diff
@@ -1,42 +1,43 @@
 {
   "images" : [
     {
-      "filename" : "app-icon-40pt.png",
+      "filename" : "Icon-App-20x20@2x-1.png",
```

Comment:
I wonder why this was changed too (given it's WP) 🤔

Reply:
This was from a change I made in a release branch – something must've gone weird in a merge, sorry about that. It's been removed.